backend/src/database/user-content/user-content.service.ts

```diff
@@ -280,11 +283,7 @@ export class UserContentService {
     if (sortOptions.sortDirection === SortDirection.desc) {
       array.reverse();
     }
-
-    array.splice(0, sortOptions.offset);
-    // decrease limit by 1 because the array starts at index 0
-    sortOptions.limit--;
-    array.splice(sortOptions.limit, array.length - sortOptions.limit);
+    array = array.slice(sortOptions.offset, sortOptions.limit);
```

Comment:
change to splice

Reply:
There was another alternative to fix the limit and offset conflict.
It is still array.slice but with an end index of (offset + limit)